### ISSUE TITLE
关于训练loss的疑问

### ISSUE THREAD

**Issue by Tongjiaxun** [CurryChen77/RIFT#8]
你好，有个问题想请教一下，如果轨迹都通过预训练充分训练过，那生成的多模态轨迹理论上reward都不会很低，那训练的loss曲线是不是不会有明显下降趋势，更多是一个震荡的状态。如果有loss曲线，可以公开一下吗？

**Comment by CurryChen77** [comment]
Hello, in our experiments, the trajectories generated by the pre-trained model are indeed of high quality. However, after migrating to the Carla simulator, the reliability of the associated probability values differs significantly from those in the dataset due to the covariate shift problem. As a result, the **trajectory-level importance sampling ratios** show large variations. This is also the reason why we adopted the **dual-clip** technique in our paper.

As for the trajectory-level reward, we use group-level advantage as defined in GRPO and apply normalization, which helps maintain the stability of the advantage values. The final loss is determined jointly by the advantage and the ratio. Therefore, even with pre-training, the initial loss after migration is not particularly low, and in our experiments, the loss generally decreases steadily. The model weights and training logs will be released within a month.
